doc/modules/language-guide/examples/grammar.txt

```diff
@@ -298,7 +298,7 @@
     <obj_body>
 
 <imp> ::= 
-    'import' <id>? '='? <text>
+    'import' <pat_nullary> '='? <text>
```

Comment:
Is there any reason this isn't just `<pat>` as for `let`?

Reply:
We don't want `(?(#foo))` here, do we?

Reply:
Yes, but I don't think we should enforce typing properties using the grammar. 
Also, it keeps the grammar closer to the informal language reference.
Plus, you might want a type annotation on the pattern without having to parenthesize it.

Reply:
Can I do this tomorrow in a new PR? I am totally burnt out, sorry.

Reply:
Of course.

Reply:
Tomorrow is saturday. Monday!

Reply:
It doesn't work: https://github.com/dfinity/motoko/pull/3076#discussion_r795963018